components/mpas-ocean/src/shared/mpas_ocn_vel_self_attraction_loading.F

```diff
@@ -0,0 +1,2396 @@
+        LoveDat(1:4,1418) = (/ 1417.0_RKIND, -6.1304462000_RKIND, 1.2921933000e-3_RKIND, -1.8595151000e-3_RKIND /)
+        LoveDat(1:4,1419) = (/ 1418.0_RKIND, -6.1307241000_RKIND, 1.2914202000e-3_RKIND, -1.8583165000e-3_RKIND /)
+        LoveDat(1:4,1420) = (/ 1419.0_RKIND, -6.1310010000_RKIND, 1.2906478000e-3_RKIND, -1.8571193000e-3_RKIND /)
+        LoveDat(1:4,1421) = (/ 1420.0_RKIND, -6.1312770000_RKIND, 1.2898761000e-3_RKIND, -1.8559234000e-3_RKIND /)
+        LoveDat(1:4,1422) = (/ 1421.0_RKIND, -6.1315521000_RKIND, 1.2891050000e-3_RKIND, -1.8547288000e-3_RKIND /)
+        LoveDat(1:4,1423) = (/ 1422.0_RKIND, -6.1318263000_RKIND, 1.2883345000e-3_RKIND, -1.8535355000e-3_RKIND /)
+        LoveDat(1:4,1424) = (/ 1423.0_RKIND, -6.1320995000_RKIND, 1.2875647000e-3_RKIND, -1.8523435000e-3_RKIND /)
+        LoveDat(1:4,1425) = (/ 1424.0_RKIND, -6.1323718000_RKIND, 1.2867956000e-3_RKIND, -1.8511528000e-3_RKIND /)
+        LoveDat(1:4,1426) = (/ 1425.0_RKIND, -6.1326432000_RKIND, 1.2860271000e-3_RKIND, -1.8499635000e-3_RKIND /)
+        LoveDat(1:4,1427) = (/ 1426.0_RKIND, -6.1329136000_RKIND, 1.2852592000e-3_RKIND, -1.8487754000e-3_RKIND /)
+        LoveDat(1:4,1428) = (/ 1427.0_RKIND, -6.1331832000_RKIND, 1.2844921000e-3_RKIND, -1.8475887000e-3_RKIND /)
+        LoveDat(1:4,1429) = (/ 1428.0_RKIND, -6.1334518000_RKIND, 1.2837255000e-3_RKIND, -1.8464033000e-3_RKIND /)
+        LoveDat(1:4,1430) = (/ 1429.0_RKIND, -6.1337196000_RKIND, 1.2829597000e-3_RKIND, -1.8452191000e-3_RKIND /)
+        LoveDat(1:4,1431) = (/ 1430.0_RKIND, -6.1339864000_RKIND, 1.2821945000e-3_RKIND, -1.8440363000e-3_RKIND /)
+        LoveDat(1:4,1432) = (/ 1431.0_RKIND, -6.1342523000_RKIND, 1.2814299000e-3_RKIND, -1.8428548000e-3_RKIND /)
+        LoveDat(1:4,1433) = (/ 1432.0_RKIND, -6.1345173000_RKIND, 1.2806660000e-3_RKIND, -1.8416745000e-3_RKIND /)
+        LoveDat(1:4,1434) = (/ 1433.0_RKIND, -6.1347815000_RKIND, 1.2799027000e-3_RKIND, -1.8404956000e-3_RKIND /)
+        LoveDat(1:4,1435) = (/ 1434.0_RKIND, -6.1350447000_RKIND, 1.2791401000e-3_RKIND, -1.8393180000e-3_RKIND /)
+        LoveDat(1:4,1436) = (/ 1435.0_RKIND, -6.1353070000_RKIND, 1.2783782000e-3_RKIND, -1.8381416000e-3_RKIND /)
+        LoveDat(1:4,1437) = (/ 1436.0_RKIND, -6.1355685000_RKIND, 1.2776168000e-3_RKIND, -1.8369666000e-3_RKIND /)
+        LoveDat(1:4,1438) = (/ 1437.0_RKIND, -6.1358290000_RKIND, 1.2768562000e-3_RKIND, -1.8357928000e-3_RKIND /)
+        LoveDat(1:4,1439) = (/ 1438.0_RKIND, -6.1360887000_RKIND, 1.2760962000e-3_RKIND, -1.8346203000e-3_RKIND /)
+        LoveDat(1:4,1440) = (/ 1439.0_RKIND, -6.1363475000_RKIND, 1.2753368000e-3_RKIND, -1.8334492000e-3_RKIND /) !}}}
+        LoveDat(1:4,1441) = (/ 1440.0_RKIND, -6.1366054000_RKIND, 1.2745781000e-3_RKIND, -1.8322792000e-3_RKIND /)
+
+
+        if ((trim(config_ocean_run_mode) .eq. 'init') .and. & 
+            (trim(config_init_configuration) .eq. 'test_sht')) then
```

Comment:
@knbarton, when we compile this in E3SM, we do not include the init code or the init flags (beginning in 'config_init').  It looks like you use this for testing during init mode? Could we just delete the condition
```
trim(config_init_configuration) .eq. 'test_sht'
```
and always set the scaling variable to 1 for init mode?

Reply:
@mark-petersen, I added this. I'll go ahead and take the condition out, as suggested.